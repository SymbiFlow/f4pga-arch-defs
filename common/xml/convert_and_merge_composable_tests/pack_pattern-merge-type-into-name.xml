<?xml version="1.0"?>
<!--
  Convert
     <pack_pattern name="xxx" type="yyy
  to
     <pack_pattern name="yyy-xxx"

  The idea is that eventually VPR will be explicit about chain and molecule
  patterns.

  -->
<xml>
 <pb_type>
  <direct>
   <pack_pattern name="B" type="chain" />
  </direct>
  <mux>
   <pack_pattern name="A" type="molecule" >
    <innertag />
   </pack_pattern>
  </mux>
 </pb_type>
</xml>
