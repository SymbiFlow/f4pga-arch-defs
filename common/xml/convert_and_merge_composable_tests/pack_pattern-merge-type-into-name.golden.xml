<?xml version="1.0"?>
<xml>
  <pb_type>
    <direct>
      <pack_pattern name="chain-B"/>
    </direct>
    <mux>
      <pack_pattern name="molecule-A">
        <innertag/>
      </pack_pattern>
    </mux>
  </pb_type>
</xml>
